react-es6-popup/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-es6-popup-example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack -w -v --display-error-details --progress --colors",
+    "start": "web-ext run -s extension/"
+  },
+  "author": "",
+  "license": "MPL-2.0",
+  "devDependencies": {
+    "babel-core": "^6.20.0",
```

Comment:
Nit: there aren't a lot of npm dependencies and they are probably following the semver convention really well, and so it is probably not a big issue, but it could make sense to use exact versions for the npm dependencies so that we are absolutely sure that the example will not break accidentally because of potential incompatible changes in a minor version update of these dependencies.